src/components/pages/CaseTable/CaseTable.js

```diff
@@ -2,38 +2,35 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import { Link, useParams } from 'react-router-dom';
 import Plot from 'react-plotly.js';
-import Tabs from '@material-ui/core/Tabs';
-import Tab from '@material-ui/core/Tab';
-import AppBar from '@material-ui/core/AppBar';
-// import Typography from '@material-ui/core/Typography';
-import Box from '@material-ui/core/Box';
-import { makeStyles } from '@material-ui/core/styles';
-
+import Highlighter from 'react-highlight-words';
 import {
-  DataGrid,
-  GridColumnsToolbarButton,
-  GridToolbarExport,
-} from '@material-ui/data-grid';
+  SearchOutlined,
+  FileTextOutlined,
+  FilePdfOutlined,
+} from '@ant-design/icons';
+import FeatherIcon from 'feather-icons-react';
 
 import {
+  Table,
+  Space,
   Button,
-  Typography,
-  Drawer,
   Input,
-  Card,
+  Radio,
+  Divider,
   Menu,
-  Dropdown,
   message,
+  Tabs,
 } from 'antd';
 import './CaseTable.less';
-
-import FeatherIcon from 'feather-icons-react';
-
-import PDFViewer from '../PDFViewer/PDFViewer';
-import PDFExportButton from './PDFOverviewExport/PDFExportButton';
+import { MenuItem } from '@material-ui/core';
```

Comment:
Is it possible to remove thus MUI element and use AntD instead?

Reply:
it should be gone. i swear i deleted this in my code IDK what happened.  delete away